README.md

```diff
@@ -103,7 +103,7 @@ A series of completed experiments exist within folders in `structured_experiment
 
```

Comment:
☝️ you need to also change the osx instructions as well since that's also pointed at Template

Reply:
Thanks I went through and fixed that and I noticed the Windows file location was not changed in the instructions either so I fixed that too.